src/openforms/accounts/models.py

```diff
@@ -63,3 +63,8 @@ def get_full_name(self):
     def get_short_name(self):
         "Returns the short name for the user."
         return self.first_name
+
+    def get_groups(self):
+        return list(self.groups.all())
```

Comment:
I cannot imagine this looks nice in the admin - this should just be a comma-separated list. Furthermore, this should be defined on the ModelAdmin as a helper method that does the proper formatting.

Additionally, you must modify the admin queryset to prefetch the groups, because with large amounts of users in the admin this will spawn an excessive amount of queries in the list view.

Reply:
Admin wise it is rendered as a comma separated list.

![Screenshot 2021-09-22 at 16 36 45](https://user-images.githubusercontent.com/60747362/134364465-68ac0f77-2633-4c86-b2b0-bd14ec9c4ae9.png)

Do you still want this be a ModelAdmin helper method?

I will modify the admin queryset to prefetch the groups though.

Reply:
> Do you still want this be a ModelAdmin helper method?

Yes - it is only relevant for the admin so let's not pollute the models with that. Additionally, that allows you to see the prefetch query & usage closely together instead of it being spread out over multiple files.